.gitlab/deps_build.yml

```diff
@@ -19,14 +19,18 @@
   script:
     # use tmpdir to prevent git remote capture by clang build
     - mkdir /tmp/clangbuild && cd /tmp/clangbuild
-    - CLANG_SHA256SUM=73f572c2eefc5a155e01bcd84815751d722a4d3925f53c144acfb93eeb274b4d
-    - LLVM_SHA256SUM=ccd87c254b6aebc5077e4e6977d08d4be888e7eb672c6630a26a15d58b59b528
-    - wget ${GENERAL_ARTIFACTS_CACHE_BUCKET_URL}/llvm/clang-11.0.1.src.tar.xz -O clang.src.tar.xz
-    - wget ${GENERAL_ARTIFACTS_CACHE_BUCKET_URL}/llvm/llvm-11.0.1.src.tar.xz -O llvm.src.tar.xz
+    - CLANG_SHA256SUM=f85c55f192cb35b9d6af1f998b7d5abe90e369f8b16445503ffab1b02e75a6f0
+    - LLVM_SHA256SUM=1e09e8c26e1b67bc94a128b62e9b9c24b70c697a2436a479c9e5eedc4ae29654
+    - wget ${GENERAL_ARTIFACTS_CACHE_BUCKET_URL}/llvm/clang-14.0.3.src.tar.xz -O clang.src.tar.xz
+    - wget ${GENERAL_ARTIFACTS_CACHE_BUCKET_URL}/llvm/llvm-14.0.3.src.tar.xz -O llvm.src.tar.xz
     - echo -e "${LLVM_SHA256SUM} llvm.src.tar.xz\n${CLANG_SHA256SUM} clang.src.tar.xz" | sha256sum --check -
     - mkdir clang && tar xf clang.src.tar.xz --strip-components=1 --no-same-owner -C clang
     - mkdir llvm && tar xf llvm.src.tar.xz --strip-components=1 --no-same-owner -C llvm
     - mkdir build && cd build
+    # copy llvm cmake modules into the correct directory to ensure discovery
+    # Build runs into this issue if the correct modules cannot be discovered:
+    # https://github.com/llvm/llvm-project/issues/53281
+    - cp ../llvm/Modules/* ../llvm/cmake/modules || echo "LLVM common cmake utils not found in directory 'Modules'"
```

Comment:
I am not sure if this is the best way to solve this problem.
Passing a flag to cmake which includes ./Modules directory in the list of include_directories is another option.
We can also patch CMakeLists.txt and explicitly add ./Modules as a directory to search.